doc/apple2enh.sgml

```diff
@@ -517,6 +517,14 @@ url="ca65.html" name="assembler manual">.
 </descrip><p>
 
 
+<sect1>Specifying file types for fopen<p>
+
+See section 
+<url url="apple2.html#ss9.4" name="Specifying file types for fopen">
+in the apple2 docoumentation.
```

Comment:
I don't like copying code or documentation because it increases the amount of work required for changes and increases the possibilities for inconsistencies when you update one and forget about the other. But, I will do it if that's they way you want it.